lxd/db/cluster/certificates.go

```diff
@@ -280,7 +284,12 @@ func DeleteCertificates(ctx context.Context, tx *sql.Tx, name string, certificat
 		return DeleteIdentitys(ctx, tx, name, api.IdentityTypeCertificateServer)
 	}
 
-	return DeleteIdentitys(ctx, tx, name, api.IdentityTypeCertificateMetrics)
+	err := DeleteIdentitys(ctx, tx, name, api.IdentityTypeCertificateMetricsRestricted)
```

Comment:
Can we add an else if here too so we dont potentially delete metrics certs if an unknown certificateType is passed in.

Reply:
Absolutely, wasn't thinking about this case.